NuRadioMC/test/atmospheric_Aeff/1e18eV/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+python T01generate_event_list.py
+python T02RunSimulation.py 1e18_full.hdf5  ../dipole_100m.json ../config.yaml output.hdf5
+python T03check_output.py
+rm tables/*txt
```

Comment:
can you add deleting the other outputfiles as well?

Reply:
Done.